include/open62541/config.h.in

```diff
@@ -65,6 +65,7 @@
 #define UA_VALGRIND_INTERACTIVE_INTERVAL ${UA_VALGRIND_INTERACTIVE_INTERVAL}
 #cmakedefine UA_GENERATED_NAMESPACE_ZERO
 #cmakedefine UA_ENABLE_PUBSUB_CUSTOM_PUBLISH_HANDLING
+#cmakedefine UA_ENABLE_PUBSUB_ETH_UADP_XDP
```

Comment:
We didn't require this global definition for ETF.
I don't see why we would for XDP.